scripts/gcp/create_host_image.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+# Copyright (C) 2024 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Create GCP image from https://github.com/google/android-cuttlefish host
+# images.
+#
+# Image Name Format: cf-debian11-amd64-<YYYYMMDD>-<commit>
+#
+# IMPORTANT!!! Artifact download URL only work for registered users (404 for guests)
+# https://github.com/actions/upload-artifact/issues/51
+
+set -e
+
+usage() {
+  echo "usage: $0 -s head-commit-sha -t /path/to/github_auth_token.txt -b bucket-name -p project-name"
+}
+
+commit_sha=
+github_auth_token_filename=
+gce_bucket=
+image_dest_project=
+
+while getopts ":hs:t:p:b:" opt; do
+  case "${opt}" in
+    h)
+      usage
+      exit 0
+      ;;
+    s)
+      commit_sha="${OPTARG}"
+      ;;
+    t)
+      github_auth_token_filename="${OPTARG}"
+      ;;
+    b)
+      gce_bucket="${OPTARG}"
+      ;;
+    p)
+      image_dest_project="${OPTARG}"
+      ;;
+    \?)
+      echo "Invalid option: ${OPTARG}" >&2
+      usage
+      exit 1
+      ;;
+    :)
+      echo "Invalid option: ${OPTARG} requires an argument" >&2
+      usage
+      exit 1
+      ;;
```

Comment:
nit: These two could be combined into
```
    \?)
      usage; exit 1
```
If you remove the starting `:` from the option string. The default messages printed by `getopts` to stderr are very similar to yours.

Reply:
Done.